libs/Microsoft.MixedReality.WebRTC.Native/src/peer_connection.cpp

```diff
@@ -880,12 +1175,25 @@ void PeerConnectionImpl::OnDataChannelAdded(
     auto lock = std::scoped_lock{data_channel_added_callback_mutex_};
     auto added_cb = data_channel_added_callback_;
     if (added_cb) {
-      DataChannelHandle data_native_handle = (void*)&data_channel;
+      mrsDataChannelHandle data_native_handle = (void*)&data_channel;
       added_cb(interop_handle, data_native_handle);
     }
   }
 }
 
+void PeerConnectionImpl::OnStreamChanged(
+    rtc::scoped_refptr<webrtc::MediaStreamInterface> stream) noexcept {
+  webrtc::AudioTrackVector audio_tracks = stream->GetAudioTracks();
+  webrtc::VideoTrackVector video_tracks = stream->GetVideoTracks();
+  RTC_LOG(LS_INFO) << "Media stream #" << stream->id()
+                   << " changed: " << audio_tracks.size()
+                   << " audio tracks and " << video_tracks.size()
+                   << " video tracks.";
+  // for (auto&& audio_track : remote_audio_tracks_) {
```

Comment:
Delete, or add TODO if something has to be added here.

Reply:
Fixed.